src/libs/actions/Report.js

```diff
@@ -1266,6 +1266,28 @@ function saveReportActionDraft(reportID, reportActionID, draftMessage) {
     Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}`, draftMessage);
 }
 
+/**
+ * Syncs up a chat report and an IOU report in Onyx after an IOU transaction has been made
+ * and they've both been updated in the back-end.
+ * @param {Object} chatReportStuff
+ * @param {Object} iouReportStuff
+ */
+function syncChatAndIOUReports(chatReportStuff, iouReportStuff) {
+    const iouReportData = {};
+    const chatReportData = {};
+    const chatReportKey = `${ONYXKEYS.COLLECTION.REPORT}${chatReportStuff.reportID}`;
+    const iouReportKey = `${ONYXKEYS.COLLECTION.REPORT_IOUS}${iouReportStuff.reportID}`;
+
+    chatReportData[chatReportKey] = getSimplifiedReportObject(chatReportStuff);
+    chatReportData[chatReportKey].iouReportID = iouReportStuff.reportID;
+    chatReportData[chatReportKey].hasOutstandingIOU = iouReportStuff.stateNum
+        === CONST.REPORT.STATE_NUM.PROCESSING && iouReportStuff.total !== 0;
```

Comment:
```suggestion
    chatReportData[chatReportKey].hasOutstandingIOU = iouReportStuff.stateNum
        === (CONST.REPORT.STATE_NUM.PROCESSING && iouReportStuff.total !== 0);
```

Let's put it into parantheses to make it clear these two conditional belong together.